Resources/Prototypes/_NF/Loadouts/Jobs/Contractor/neck.yml

```
@@ -339,16 +339,16 @@
   effects:
   - !type:GroupLoadoutEffect
     proto: ContractorT3
-  price: 2500
+  price: 1500
```

```suggestion
  price: 2500
```
Revert.  If T3 items cost 2500, and this is a T3 item, it should cost 2500.

---

I feel like the cloaks are a bit expensive, but I guess that's an issue for another PR

Also, aren't the T1, T2, T3 only meant as time requirements ? It seems a bit odd to me that they have tp cost more just because they're "higher tier"